lib/AST/NameLookup.cpp

```diff
@@ -662,6 +662,22 @@ static void recordShadowedDeclsAfterTypeMatch(
         }
       }
 
+      // Next, prefer any other module over the _Backtracing module.
+      if (auto spModule = ctx.getLoadedModule(ctx.Id_Backtracing)) {
+        if ((firstModule == spModule) != (secondModule == spModule)) {
+          // If second module is _StringProcessing, then it is shadowed by
```

Comment:
This is right. Would be nice if there was a little local function to clean up some of the duplicated parts of this check-and-shadow logic for each module.